src/main/java/com/gluonhq/substrate/config/ConfigResolver.java

```diff
@@ -61,11 +64,35 @@
 
     private final List<File> jars;
 
-    public ConfigResolver(String classpath) {
-        this.jars = new ClassPath(classpath)
+    /**
+     * ConfigResolver constructor
+     *
+     * @param classpath a string with the full classpath of the user's project
+     * @param tmpPath a temporal path where classes.jar will be added
```

Comment:
is tmpPath needed outside of this method? If not, it seems better to create/delete it inside this method?

Reply:
We just need a temporal path, and using `client/archOS/gvm/tmp` seems convenient, but then we need to pass that path.

Reply:
Don't we have Paths.getTmpPath or so?

Reply:
No, we don't. Well, we could use Files.createTempDirectory, outside client/archOS/gvm

Reply:
If we don't need the results outside of this method, Files.createTempDirectory sounds better.